src/tests/test_module.py

```diff
@@ -519,6 +519,18 @@ def test_incrby(self):
             assert result[0][0] >= start_incr_time
             assert len(result) <= 40
 
+    def test_incrby_with_timestamp(self):
+        with self.redis() as r:
+            r.execute_command('ts.create', 'tester')
+
+            for i in range(20):
+                assert r.execute_command('ts.incrby', 'tester', '5', 'TIMESTAMP', i) == 'OK'
```

Comment:
@ashtul can you please also add test with RESET and batch events in the same reset time-backet?

Reply:
Done